core/src/test/resources/custom-fields.adoc

```diff
@@ -86,10 +86,10 @@ mutation { createPerson(name:"Jill") {name} }
 .Cypher
 [source,cypher]
 ----
-CALL apoc.cypher.doIt('WITH $name AS name CREATE (p:Person { name: \"Test\"+$name }) RETURN p',  {
-	name: $createPersonName
-}) YIELD value
-WITH value[head(keys(value))] AS createPerson
+CALL {
+	WITH $createPersonName AS name
```

Comment:
we need to watch out here for naming conflicts if we rename to a single/simple name, we should still be able to use parameters in the subqueries

Reply:
I see no potential conflict here:

- `$createPersonName` is defined by the library
- `name` is the name of the fields argument defined by the user